ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -108,20 +108,21 @@ RUN mkdir /tmp/hub \
  && sudo ./install \
  && rm -rf /tmp/hub
 
-# Install the latest Node.js and npm.
-# Non-sudo global packages: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md
-RUN git clone https://github.com/nodejs/node /tmp/node \
- && cd /tmp/node \
- && git checkout v8.2.1 \
- && ./configure \
- && make -j18 \
- && sudo make install \
- && rm -rf /tmp/node \
+# Install the latest nvm
+RUN mkdir /tmp/nvm \
+ && cd /tmp/nvm \
+ && wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash \
```

Comment:
Nit: You're creating a `/tmp/nvm` directory, but you're not actually using it because you pipe the `wget` directly into `bash`. However, are you sure you don't need `sudo` to install NVM?

Maybe it would be better to wget the `install.sh` script into the directory you created, and then run it with `sudo`?